features/support/publishing_cuke_helpers.rb

```diff
@@ -26,6 +26,8 @@ def make_post(text)
   def submit_publisher
     txt = find("#publisher #status_message_text").value
     find("#publisher .btn-primary").click
+    # wait for the publisher to be closed
+    find("#publisher")["class"].should include("closed")
```

Comment:
We add the class [when the publisher should be opened on page load](https://github.com/diaspora/diaspora/blob/55fee86a95665446b197e16606be81378664e773/app/views/publisher/_publisher.html.haml#L1). (However the class is not needed anymore)

Maybe the getting started popovers are a problem. You could close them before doing anything with the publisher (like submitting the publisher).

```
And I wait for the popovers to appear
And I click close on all the popovers
```